### PR TITLE
fix(css): stop the endless css rebuilds

### PR DESCRIPTION
**Pull request by dannycoates**

### CODE REVIEW

File: packages/fxa-settings/pm2.config.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ module.exports = {
         'src/components/**/*.scss',
         require.resolve('fxa-react/configs/tailwind.js'),
       ],
-      ignore_watch: ['src/styles/tailwind.out.css'],
+      ignore_watch: ['src/styles/tailwind.out.*'],
```
Comment (jodyheavener) [inline packages/fxa-settings/pm2.config.js:42]: Might be worth adding this to the other tailwind projects since they may change to SCSS one day

Comment (dannycoates) [reply packages/fxa-settings/pm2.config.js:42]: go ahead ;)

Comment (jodyheavener) [reply packages/fxa-settings/pm2.config.js:42]: lol fair. clear to merge this

Comment (jodyheavener) [reply packages/fxa-settings/pm2.config.js:42]: lol fair. clear to merge this

**Pull request merged**